stl/inc/format

```diff
@@ -3961,6 +3961,13 @@ _NODISCARD size_t formatted_size(const locale& _Loc, const wformat_string<_Types
 _FMT_P2286_END
 
 #if _HAS_CXX23
+template <class _CharT>
+_NODISCARD int _Measure_display_width(const basic_string_view<_CharT> _Value) {
+    int _Width = -1;
+    (void) _Measure_string_prefix(_Value, _Width);
+    return _Width;
+}
+

```

Comment:
I'd prefer if this was inlined, it's really a simple wrapper and it only saves one line in each caller, in exchange for the 6-line function definition (not that code-golf is something we optimize for). I just think it doesn't increase clarity to redirect through this vs just calling _Measure_string_prefix directly.

Reply:
oh, it's only used once too. Definitely inline this.

Reply:
I added this wrapper because:

1. The same functionality is needed by `range_formatter` (#4642) and `formatter<std::filesystem::path>`, and might be needed by new formatters in the future.
2. The name `_Measure_string_prefix` is IMO misleading here: it's measuring the entire string, not a (proper) prefix.
3. I'd also prefer to avoid uses of out parameters.

Reply:
That rationale seems compelling to me (particularly adapting the out-param to a normal return type).

Reply:
I can live with it. I still prefer the function be inlined but if it's used more than once it's more stylistic.